test ci trigger

Zeref996 opened the pull request:
test ci

nepeplwu committed:
Add module v2

nepeplwu committed:
fix bug

nepeplwu committed:
fix bug

nepeplwu committed:
Fix bug

nepeplwu committed:
fix bug

nepeplwu committed:
Merge branch 'develop' of https://github.com/PaddlePaddle/PaddleHub into develop

nepeplwu committed:
Fix bug

nepeplwu committed:
Fix bug

nepeplwu committed:
fix bug

nepeplwu committed:
Merge branch 'develop' of https://github.com/PaddlePaddle/PaddleHub into develop

nepeplwu committed:
Fix bug

nepeplwu committed:
Merge branch 'develop' of https://github.com/PaddlePaddle/PaddleHub into develop

nepeplwu committed:
Merge branch 'develop' of https://github.com/PaddlePaddle/PaddleHub into develop

Leowolfking committed:
add new documents (#774)

* add new documents

* add picture

haoyuying committed:
hrnet_w18_samll_v1_humanseg and shufflenet_humanseg (#778)

ShenYuhan committed:
delete serving index.html (#802)

ShenYuhan committed:
print traceback for serving (#811)

ShenYuhan committed:
modify lac serving demo. (#824)

* modify lac serving demo.

* update lac_serving_demo.py

ShenYuhan committed:
fix gbk bug on windows (#836)

haoyuying committed:
Add sr (#808)

T-baby committed:
Delete some duplicate codes (#832)

- Delete some duplicate codes
- Fix the problem of not being able to process unlogged words

moguguo committed:
Update more_demos.md (#850)

wulei123456789 committed:
update docs

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/PaddlePaddle/PaddleHub?pullRequest=1265) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://cla-assistant.io/PaddlePaddle/PaddleHub?pullRequest=1265) before we can accept your contribution.<br/>**5** out of **6** committers have signed the CLA.<br/><br/>:white_check_mark: moguguo<br/>:white_check_mark: haoyuying<br/>:white_check_mark: T-baby<br/>:white_check_mark: Leowolfking<br/>:white_check_mark: ShenYuhan<br/>:x: wulei123456789<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/PaddlePaddle/PaddleHub?pullRequest=1265) it.</sub>

haoyuying committed:
Add deeplabv3 and hrnetw18.

nepeplwu committed:
Add the processing of PaddlePaddle development version.

nepeplwu committed:
Fix the problem that ModuleV1 fails to export inference model.

nepeplwu committed:
Add an interface for exporting onnx format model.

nepeplwu committed:
Print the miou in the segmentation training demo.

KPatr1ck committed:
Add audio classification module and ESC50 dataset.

nepeplwu committed:
Improve the model export interface.

nepeplwu committed:
Add save_inference_model api.

nepeplwu committed:
Fixed the problem that the training cannot be resumed when there is no metric file in the checkpoint.

haoyuying committed:
Update README.md

haoyuying committed:
Update README.md

chunzhang-hub committed:
Update QR code.

nepeplwu committed:
Fix the problem that the `processing` interface does not exit when the main thread crash.

nepeplwu committed:
Adjust the order of pip import.

nepeplwu committed:
Add `ignore_env_mismatch` in hub.Module.

haoyuying committed:
Raise an exception when the specified module does not support evaluation.

KPatr1ck committed:
Add README.md and requirements.txt in nlp&audio modules.

nepeplwu committed:
Merge branch 'release/v2.0' into release/v2.1

nepeplwu committed:
Add author.

nepeplwu committed:
Update releaes note.

haoyuying committed:
Fix the prediction problem of semantic segmentation module.

KPatr1ck committed:
Update audio README.md.

cqvu committed:
Add module serving example using BentoML.

nepeplwu committed:
Remove output logs in bentoml ipynb.

nepeplwu committed:
Add contributors in README. (#1378)

nepeplwu committed:
Add links.

DeepGeGe committed:
Fix ocr text detection module bugs.

haoyuying committed:
Fix the problem that the colorization model reads data incorrectly on Paddle 2.1.

nepeplwu committed:
Update docs.

chunzhang-hub committed:
Update contributor information and model information.

nepeplwu committed:
Fix documentation errors.

KPatr1ck committed:
Add UnifiedTransformer.

KPatr1ck committed:
Update plato2_en_base and plato2_en_large README.md.

D-DanielYang committed:
Update readme concisely.

D-DanielYang committed:
Update README_ch.md

D-DanielYang committed:
Update README_ch.md

D-DanielYang committed:
Update README_ch.md

D-DanielYang committed:
Update README_ch.md

D-DanielYang committed:
Update README_ch.md

D-DanielYang committed:
Update README_ch.md

haoyuying committed:
Fix image classification docs typo.

paopjian committed:
Fix link issue.

nepeplwu committed:
Fix travis-ci issue.

nepeplwu committed:
Update travis config.

nepeplwu committed:
Update docs.

D-DanielYang committed:
Update README_ch.md

D-DanielYang committed:
Update README.md

nepeplwu committed:
Merge branch 'release/v2.1' into develop

nepeplwu committed:
Merge branch 'release/v2.1' into develop

nepeplwu committed:
Add alias for paddle.hub.*

haoyuying committed:
Add more semantic segmentation models

D-DanielYang committed:
test=release/v2.1, test=documents_fix

D-DanielYang committed:
Merge pull request #1447 from grasswolfs/update_joinus_0607

update the joinus

D-DanielYang committed:
test=release/v2.1, test=documents_fix

D-DanielYang committed:
Merge pull request #1448 from grasswolfs/update_readmeen_0607

update the readme_en

D-DanielYang committed:
Update README.md

D-DanielYang committed:
Update README_ch.md

D-DanielYang committed:
Update README.md

nepeplwu committed:
Merge branch 'develop' of https://github.com/PaddlePaddle/PaddleHub into develop

nepeplwu committed:
Fixed the issue that the model does not support training in multi-card scene.

chunzhang-hub committed:
Update README_ch.md

新增对panpjian同学修改模型链接的错误问题

chunzhang-hub committed:
Update README.md

新增感谢信息：外部开发者对模型链接的修正

nepeplwu committed:
Add rexnet.

nepeplwu committed:
Merge branch 'develop' into rexnet

KPatr1ck committed:
Correct import snippet in some READMEs

D-DanielYang committed:
Merge pull request #1453 from chunzhang-hub/release/v2.1

update_thanksinfo

D-DanielYang committed:
Merge pull request #1454 from nepeplwu/rexnet

Add RexNet.

D-DanielYang committed:
Merge pull request #1456 from KPatr1ck/release/v2.1

Correct import snippet in some READMEs

D-DanielYang committed:
Update installation.rst

D-DanielYang committed:
Update README.md

D-DanielYang committed:
Update README.md

D-DanielYang committed:
Update README.md

D-DanielYang committed:
Update README_ch.md

D-DanielYang committed:
Update README.md

KPatr1ck committed:
Add machine translation model

D-DanielYang committed:
Update README_ch.md

nepeplwu committed:
Add GhostNet.

nepeplwu committed:
Add HRNet.

nepeplwu committed:
Add RepVGG.

dependabot[bot] committed:
Bump nltk in /modules/text/machine_translation/transformer/en-de

dependabot[bot] committed:
Bump nltk in /modules/text/machine_translation/transformer/zh-en

KPatr1ck committed:
Update dependencies of machine translation modules

KPatr1ck committed:
Solve code formatting problem for ci

KPatr1ck committed:
Fix hub serving config in lac module.

KPatr1ck committed:
Add voice cloning module: lstm_tacotron2

nepeplwu committed:
Fix docs format issue.

haoyuying committed:
Add deeplabv3p_resnet50_voc readme

KPatr1ck committed:
Convert dtype of input tensors to int64 explicitly in plato-mini

haoyuying committed:
Fix the compatibility issue of deeplabv3p on paddlepaddle 2.1

haoyuying committed:
Fix the compatibility issue of ocrnet on paddlepaddle 2.1

ZeyuChen committed:
Update README.md

nepeplwu committed:
Add GhostNet.

KPatr1ck committed:
Update requirements and README.md of lstm_tacotron2

wj-Mcat committed:
Add return_prob in text classification module

kinghuin committed:
Update ernie_gen

haoyuying committed:
Add ocrnet_hrnetw18_voc README

nepeplwu committed:
Fix ernie_gen* bug.

kinghuin committed:
Fix ernie_gen evaluation bug

D-DanielYang committed:
Update README.md

haoyuying committed:
Update python_use_hub.rst

KPatr1ck committed:
Cache tokenizer in TransformerModule (#1491)

zbp-xxxp committed:
Update utils.py (#1508)

The binary mode of fromstring is deprecated, as it behaves surprisingly on unicode inputs. Use frombuffer instead

haoyuying committed:
add 0.0.0 version (#1521)

haoyuying committed:
Fix compatibility issues

KPatr1ck committed:
Fix ernie_gen training error in windows device (#1540)

KPatr1ck committed:
Fix compatibility issue in machine translation models (#1542)

nepeplwu committed:
Fix image classification module error

nepeplwu committed:
Merge branch 'release/v2.1' into develop

nepeplwu committed:
Update docs

nepeplwu committed:
Warn when using Chinese directory under windows.

nepeplwu committed:
Remove gitpython dependency.
